src/autoload.php

```diff
@@ -2,7 +2,10 @@
 
 namespace src;
 
-define('MOODLE_INTERNAL', 1);
+// In Moodle this is set, however in testing this is not. So need to set it if not set or leave it be.
```

Comment:
Interesting, what do you mean by "in testing"?

Reply:
I thought that is why you guys had that there? Otherwise, I don't know why the code needs a redefine for 'moodle_internal'?

Reply:
Ah sorry, yeah you're totally right.